source/basic.tex

```diff
@@ -2870,7 +2870,8 @@
 or defines a constexpr variable initialized to a TU-local value (defined below).
 \begin{note}
 An inline function template can be an exposure even though
-explicit specializations of it might be usable in other translation units.
+some
+explicit specializations of it would be usable in other translation units.
```

Comment:
Maybe "certain" instead of "some"?  We don't want to suggest that such explicit specializations always exist, merely that they, er, _could_.